### DISCUSSION TITLE
validate.run unexpected keyword argument single-cls

### DISCUSSION THREAD

**Discussion by maxPrakken** [ultralytics/yolov5#9945] [Q&A]
im trying to run Yolov5 on custom data in a custom docker container. The startup is already really slow and takes a while to get started.
![image](https://user-images.githubusercontent.com/22146912/198303520-baebd08e-444e-4133-a358-cd8b61fe410b.png)

my dataset is set up in the default directory structure according to the ultralytics documentation.

`python3 -u classify/train.py --name $(date +"%d-%m-%Y") --epochs 50 --batch-size 64 --data anon_imization.yaml --weights yolov5s.pt --cfg yolov5s.yaml"$@"` 

![image](https://user-images.githubusercontent.com/22146912/198304618-90317c46-d27c-40eb-beac-6fcee5fc8f16.png)

as soon as the first epoch is done and its starts its validation run it throws `unexpected keyword argument single-cls` Eventho that is a default parameter.

Im not sure why this happens, can anyone help me out with this?

edit: I'm working on a rtx titan 24gb VRAM

**Comment by somanand88** [comment]
> im trying to run Yolov5 on custom data in a custom docker container. The startup is already really slow and takes a while to get started. ![image](https://user-images.githubusercontent.com/22146912/198303520-baebd08e-444e-4133-a358-cd8b61fe410b.png)
> 
> my dataset is set up in the default directory structure according to the ultralytics documentation.
> 
> `python3 -u classify/train.py --name $(date +"%d-%m-%Y") --epochs 50 --batch-size 64 --data anon_imization.yaml --weights yolov5s.pt --cfg yolov5s.yaml"$@"`
> 
> ![image](https://user-images.githubusercontent.com/22146912/198304618-90317c46-d27c-40eb-beac-6fcee5fc8f16.png)
> 
> as soon as the first epoch is done and its starts its validation run it throws `unexpected keyword argument single-cls` Eventho that is a default parameter.
> 
> Im not sure why this happens, can anyone help me out with this?
> 
> edit: I'm working on a rtx titan 24gb VRAM

I got the same error: **TypeError: run() got an unexpected keyword argument 'single_cls'**
